cloudwatchlogs-with-dlq/cloudwatchlogs_lambda.js

```diff
@@ -24,44 +24,53 @@ function createRecords(config, events, awslogsData) {
     var records = [];
     var lastRequestID = null;
     console.log('Log events: ' + events.length);
-
+    var msgset = new Set();
```

Comment:
Do we need to worry about memory size for the function then? How often does this happen?